### ISSUE TITLE
Renaming a content type doesn't change the file names

### ISSUE THREAD

**Issue by locopati** [strapi/strapi#5742]
**Describe the bug**
When you rename a content type, the file names are still the same as the original type name

**Steps to reproduce the behavior**
1. In the UI, rename a content type
2. Look at the file names in api/<old-content-type> and in api/<old-content-type>/config/routes.json

**Expected behavior**
The files and routes and db collection (in mongo) are renamed to the new content type name

**Additional context**
Since the UI doesn't do it, one would have to do a lot of manual work to change this over in order to have consistency between the UI naming and the file naming

**Comment by derrickmehaffy** [comment]
I will leave this one open even though it doesn't have the complete bug report template simply because I know it doesn't do this, and never has.

**Comment by alexandrebodin** [comment]
This is the intended behaviour as doing file renaming changes the name of the content type in the codebase and can break your code or database. The renaming is a simple display name change. If you to do renaming you should do this by hand to control the database names and such.

**Comment by drazik** [comment]
I understand that renaming a content type from the UI doesn't rename files, database and API names. But is it possible to manually rename a content type and its database field / API name ?

**Comment by Pallavi-KS** [comment]
Facing same issue as I have to modify the name of content type and its related fields names like DB field, Api name mainly Singular name and Plural name. 
I was able to change the collection name by doing rename in DB(Using Postgres) , and changing "collectionName" to connect with same.
 This end me having error error: The key of the content-type should be the same as its singularName. Found XYZ and ABC.
**Error: The key of the content-type should be the same as its singularName. Found XYZ and ABC.
    at validateKeySameToSingularName**

Am I missing anything in this process of updating collection name every where?
